WebHostLib/customserver.py

```diff
@@ -171,8 +170,8 @@ def get_save(self) -> dict:
         return d
 
 
-def get_random_port():
-    return random.randint(49152, 65535)
+def get_random_port(room_ports_config: dict):
```

Comment:
Type hint for dict takes key and value types
```suggestion
def get_random_port(room_ports_config: dict[str, typing.Any]):
```